.ci/pipeline_definitions

```diff
@@ -3,10 +3,54 @@
 # SPDX-License-Identifier: Apache-2.0
 
 gardener-extension-registry-cache:
+  templates:
+    helmcharts:
+    - &registry-cache
+      name: registry-cache
+      dir: charts/gardener-extension-registry-cache
+      registry: europe-docker.pkg.dev/gardener-project/snapshots/charts/gardener/extensions
+      mappings:
+      - ref: ocm-resource:gardener-extension-registry-cache.repository
+        attribute: image.repository
+      - ref: ocm-resource:gardener-extension-registry-cache.tag
+        attribute: image.tag
+    - &registry-cache-admission-application
+      name: admission-registry-cache-application
+      dir: charts/admission/charts/application
+      registry: europe-docker.pkg.dev/gardener-project/snapshots/charts/gardener/extensions
+      mappings:
+      - ref: ocm-resource:gardener-extension-registry-cache-admission.repository
+        attribute: global.image.repository
+      - ref: ocm-resource:gardener-extension-registry-cache-admission.tag
+        attribute: global.image.tag
+    - &registry-cache-admission-runtime
+      name: admission-registry-cache-runtime
+      dir: charts/admission/charts/runtime
+      registry: europe-docker.pkg.dev/gardener-project/snapshots/charts/gardener/extensions
+      mappings:
+      - ref: ocm-resource:gardener-extension-registry-cache-admission.repository
+        attribute: global.image.repository
+      - ref: ocm-resource:gardener-extension-registry-cache-admission.tag
+        attribute: global.image.tag
+
   base_definition:
     traits:
       version:
         preprocess: 'inject-commit-hash'
+      publish:
+        oci-builder: 'docker-buildx'
+        platforms:
+        - linux/amd64
+        - linux/arm64
+        dockerimages:
+          gardener-extension-registry-cache:
+            image: europe-docker.pkg.dev/gardener-project/snapshots/gardener/extensions/registry-cache
+            dockerfile: 'Dockerfile'
+            target: registry-cache
+          gardener-extension-registry-cache-admission:
+            image: europe-docker.pkg.dev/gardener-project/snapshots/gardener/extensions/registry-cache-admission
+            dockerfile: 'Dockerfile'
+            target: registry-cache-admission
   jobs:
     head-update:
```

Comment:
Can we also enable cleanup of snapshots?

```diff
diff --git a/.ci/pipeline_definitions b/.ci/pipeline_definitions
index 2286c92b..4012315e 100644
--- a/.ci/pipeline_definitions
+++ b/.ci/pipeline_definitions
@@ -58,6 +58,7 @@ gardener-extension-registry-cache:
           ocm_repository: europe-docker.pkg.dev/gardener-project/snapshots
           ocm_repository_mappings:
             - repository: europe-docker.pkg.dev/gardener-project/releases
+          retention_policy: 'clean-snapshots'
         draft_release: ~
         options:
           public_build_logs: true
```

Reply:
ever since we switched to GAR, there is automatic cleanup of any artefact uploaded to `snapshots`-registry, so explicit cleanup via pipeline is no longer needed

Reply:
Hm, okay. They maybe you can consider removing this feature from the CI/CD and cleaning up configurations of `retention_policy: 'clean-snapshots'` (there are many, see https://github.com/search?q=org%3Agardener%20clean-snapshots&type=code) as we already switched to GAR.